Makefile

```diff
@@ -0,0 +1,51 @@
+RUBY := $(shell command -v ruby 2>/dev/null)
+HOMEBREW := $(shell command -v brew 2>/dev/null)
+BUNDLER := $(shell command -v bundle 2>/dev/null)
+
+default: setup
+
+setup: \
+	pre_setup \
+	check_for_ruby \
+	check_for_homebrew \
+	install_bundler \
+	install_gems
+
+pre_setup:
+	$(info ------------------------------)
+	$(info iOS project setup ...)
+	$(info ------------------------------)
+
+check_for_ruby:
+	$(info )
+	$(info Checking for Ruby ...)
+
+ifeq ($(RUBY),)
+	$(error Ruby is not installed)
+endif
+
+check_for_homebrew:
+	$(info )
+	$(info Checking for Homebrew ...)
+
+ifeq ($(HOMEBREW),)
+	$(error Homebrew is not installed)
+endif
+
+install_bundler:
+	$(info )$(info Checking and install bundler ...)
+
+ifeq ($(BUNDLER),)
+	gem install bundler -v '~> 2.0'
+else
+	gem update bundler '~> 2.0'
+endif
+
+install_gems:
+	$(info )
+	$(info Install Ruby Gems ...)
+
+	bundle install --without=documentation
+
+danger:
+	bundle exec danger
```

Comment:
this is awesome :smile: but feels a little to heavy compared to just having a `.travis.yml`

could we just update the travis config to use `gem update bundler` and `bundle install --without=documentation` ?

Reply:
Ha, sure.  

I was expecting that 😄.  My original thought/intention was extracting the script stage logic but felt it was out of scope.  Also, wasn't sure your thoughts on that approach.  It's something I did at work to share common commands between multiple projects.